apps/utils/deletion.py

```diff
@@ -224,6 +224,15 @@ def get_related_pipelines_queryset(instance, pipeline_param_key: str = None):
     return pipelines
 
 
+def get_related_pipeline_experiments_queryset(instance, pipeline_param_key: str):
+    from apps.experiments.models import Experiment
+
+    return Experiment.objects.filter(
```

Comment:
```suggestion
    return Experiment.objects.exclude(pipeline=None).filter(
```

Reply:
Experiments without pipelines are excluded by the filter already?

Reply:
They are excluded implicitly. Its just nice to make it explicit.

Reply:
Good point. See https://github.com/dimagi/open-chat-studio/pull/1587/commits/be84bd635bf144cb3b11c13a4d0915b52ce6d77d